### ISSUE TITLE
Destination Location Tracker (or new Sensor entity) based on Tesla Navigation (not HA zones)

### ISSUE THREAD

**Issue by GitHubGoody** [alandtse/tesla#482]
**Is your feature request related to a problem? Please describe.**
The destination location tracker (device_tracker.###_destination_location_tracker) seems to be linked to the HA zones (e.g. Home, etc.) rather than the actual Tesla destination. Of course, that means that often the sensor indicates not_home instead of an actual destination.

**Describe the solution you'd like**
The destination sensor state to indicate the name (address if no name) of the current destination in the car.

**Describe alternatives you've considered**
N/A

**Additional context**
I'm submitting this as a feature request since this may be working as designed (which I suspect since it was created as a device_tracker entity). If so, I'm requesting an additional sensor to implement the solution.

**Comment by alandtse** [comment]
This is as designed given HA limitations. The sensor also has attributes that say where you're going if you need those details for scripting. I believe it's been discussed in the discussions tab.

**Comment by GitHubGoody** [comment]
Is this the Discussion you're referencing - https://github.com/alandtse/tesla/discussions/403? If so, I think they're talking about the Location Tracker. Having it tied to HA Zones makes sense. Then, also having the Destination Location Tracker work similarly also makes sense.

So, what I'm asking for is a new sensor - sensor.xxx_navigation_destination that simply states to the string (business name, address, etc.) that (hopefully) is available from the Tesla API.

**Comment by alandtse** [comment]
Yes I guess that. Again, the attribute already exists. That's not enough?

**Comment by GitHubGoody** [comment]
I did a test this afternoon after your first reply. With a destination in, I triggered an update then checked that entity's attributes. I saw the coordinates (of the destination I assume, but I didn't plot it).

But, I did not see the name or address of the destination's attributes.

**Comment by carleeno** [comment]
@GitHubGoody see https://github.com/alandtse/tesla/issues/477#issuecomment-1374686105

TL;DR: the attribute you're looking for is under the arrival time sensor.

**Comment by GitHubGoody** [comment]
Thanks. I'll check it out tomorrow and make another template sensor if it's what I'm looking for.

**Comment by GitHubGoody** [comment]
I think that attribute will work. Thank you.
